dask_expr/io/tests/test_from_pandas.py

```diff
@@ -0,0 +1,110 @@
+import pytest
+from dask.dataframe.utils import assert_eq
+
+from dask_expr import from_pandas, repartition
+from dask_expr.tests._util import _backend_library
+
+pd = _backend_library()
+
+
+@pytest.fixture(params=["Series", "DataFrame"])
+def pdf(request):
+    out = pd.DataFrame({"x": [1, 4, 3, 2, 0, 5]})
+    return out["x"] if request.param == "Series" else out
+
+
+@pytest.mark.parametrize("sort", [True, False])
+def test_from_pandas(pdf, sort):
+    df = from_pandas(pdf, npartitions=2, sort=sort)
+
+    assert df.npartitions == 2
+    assert df.divisions == (0, 3, 5) if sort else (None, None, None)
```

Comment:
Can you rebase? This test should fail for sort=False

We set divisions even if you set sort=False if your Index is already sorted

Reply:
I already rebased

Reply:
```

pdf = pd.DataFrame({"x": [1, 4, 3, 2, 0, 5]})
df = from_pandas(pdf, npartitions=2, sort=False)
assert df.divisions == (0, 3, 5)

```

this passes for me, so your test should fail

Reply:
ah it's not a rebase issue, your syntax is incorrect

Reply:
```suggestion
    assert df.divisions == ((0, 3, 5) if sort else (None, None, None))
```

This will fail